### PR TITLE
Refactor templates

### PR DESCRIPTION
**Pull request by wooorm**
### Initial checklist

* [x] I read the support docs 
* [x] I read the contributing guide 
* [x] I agree to follow the code of conduct 
* [x] I searched issues and couldn’t find anything (or linked relevant results below) 
* [x] I’ve added docs and tests or that’s not applicable

### Description of changes

I went through the templates and refactored them based on my experience with issues/PRs the last couple of years!
Tried to make the flow better.
Less verbose in some cases, more clear examples of what to fill in in others. Some more things optional.

FYI, I was inspired a bit by https://whatwg.org/faq#adding-new-features.

### CODE REVIEW

**Review by ChristianMurphy — APPROVED** [review]

File: .github/ISSUE_TEMPLATE/1-bug.yml [hunk 1]
```diff
@@ -16,82 +17,86 @@ body:
...
       required: true
   - attributes:
-      label: Affected runtime and version
+      label: Runtime
       description: |
-        What runtime are you using (such as `node@18.2` or `deno@1.12.2`)?
-        Please test using the latest version.
+        What runtime are you using?
+        Please test using latest versions.
+        Example: `node@23.2.0` or `deno@2.0.6`.
```
Comment (remcohaszing) [inline .github/ISSUE_TEMPLATE/1-bug.yml:71]: Maybe add a browser as an example? If the error happens in a browser, the Node.js version used to launch a dev server is irrelevant. I think this is a common cause of confusion.

```suggestion
        Example: `node@23.2.0`, `deno@2.0.6`, or `Google Chrome 131`
```

File: .github/ISSUE_TEMPLATE/1-bug.yml [hunk 2]
```diff
@@ -16,82 +17,86 @@ body:
...
     id: runtime
     type: input
     validations:
-      required: true
+      required: false
```
Comment (remcohaszing) [inline .github/ISSUE_TEMPLATE/1-bug.yml:75]: Why don’t we require this anymore?

File: .github/ISSUE_TEMPLATE/1-bug.yml [hunk 3]
```diff
@@ -16,82 +17,86 @@ body:
...
+      label: Operating system
     id: operating-system
     type: input
     validations:
       required: false
   - attributes:
       description: |
         What tools are used to build or bundle the project?
-        Please test using the latest version.
+        Please test using latest versions.
```
Comment (remcohaszing) [inline .github/ISSUE_TEMPLATE/1-bug.yml:99]: Nitpicky, but I think this is better. Feel free to ignore.

```suggestion
        Please test using the latest versions.
```

File: .github/ISSUE_TEMPLATE/1-bug.yml (hunk 2 above)
Comment (wooorm) [reply .github/ISSUE_TEMPLATE/1-bug.yml:75]: because they are not always relevant.

File: .github/ISSUE_TEMPLATE/1-bug.yml (hunk 1 above)
Comment (wooorm) [reply .github/ISSUE_TEMPLATE/1-bug.yml:71]: Maybe maybe.

I added a note somewhere else to prefer reproducing the problem with plain Node/npm.

The thinking goes that sometimes people provide a repo with an entire Next app, when it’s a question about how markdown compiles to HTML.

Most of our stuff works anywhere, browsers, deno/bun/node. And node is easiest for us to debug. It’s also what we base our compatibility on.

I thought about adding Bun too. But I preferred being less verbose.

We could also drop it, and do:

```suggestion
        Example: `node@23.2.0`.
```

I also tried to use the actual output of version commands right now. So, looking at my Chrome’s about info, perhaps this should be `Google Chrome 131.0.6778.86`?

I think I’d prefer my above suggestion (just node) or:

```suggestion
        Example: `node@23.2.0` or `Chrome 131.0.6778.86`.
```

File: .github/ISSUE_TEMPLATE/1-bug.yml (hunk 3 above)
Comment (wooorm) [reply .github/ISSUE_TEMPLATE/1-bug.yml:99]: /cc @ChristianMurphy as a native English speaker, strong opinions?

File: .github/ISSUE_TEMPLATE/1-bug.yml (hunk 1 above)
Comment (remcohaszing) [reply .github/ISSUE_TEMPLATE/1-bug.yml:71]: I agree with all of that, but in practice people also run into issues in other runtimes and not everyone is equally able or wanting to make a nice Node.js reproduction. Also some packages work differently in the browser or Node.js, i.e. when we specify the `browser` export condition.

Comment (wooorm) [reply .github/ISSUE_TEMPLATE/1-bug.yml:71]: right, true! any of these alternatives of mine that you 👍

Comment (remcohaszing) [reply .github/ISSUE_TEMPLATE/1-bug.yml:71]: Either is :+1: really, but I prefer to include Chrome.

**Pull request merged**